synapse/cryotank.py

```diff
@@ -242,6 +240,15 @@ def postCell(self):
             tank = CryoTank(path, conf)
             self.tanks.put(name, tank)
 
+    def initConfDefs(self):
+        super().initConfDefs()
+        self.addConfDefs((
+            ('defvals', {'defval': {},
+                'ex': '{mapsize: 1000000000}',
```

Comment:
Should this be `'ex': '{"mapsize": 1000000000}',`?

Reply:
ok